lte/gateway/c/core/oai/lib/s6a_proxy/s6a_client_api.cpp

```diff
@@ -144,6 +145,8 @@ static void s6a_handle_update_location_ans(const std::string& imsi,
       itti_msg->result.choice.base = DIAMETER_SUCCESS;
       magma::convert_proto_msg_to_itti_s6a_update_location_ans(response,
                                                                itti_msg);
+      // write to subscriberdb
+
```

Comment:
Here I need to get the apn information from response. How can I know what response looks like? How do I know what subscriber data looks like? Is this from proto? 
Do I populate the information in subscriber data by checking if the data actually exists in response.